frontend/src/components/tables/IndustryTable.tsx

```
@@ -0,0 +1,199 @@
+      render: (value, industry) => (
+        <div className="flex items-center gap-2">
+          <span className="font-bold text-sm block truncate">
+            {String(value)}
+          </span>
+          <button
+            onClick={(e) => {
+              e.stopPropagation();
+              onIndustryView?.(industry);
+            }}
+            className="p-1 rounded border border-gray-600 transition-colors text-white hover:text-gray-200 hover:bg-gray-700 hover:border-gray-500 flex-shrink-0"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)